vllm/engine/llm_engine.py

```diff
@@ -1616,7 +1616,24 @@ def _get_stats(self,
         num_generation_tokens_requests: List[int] = []
         n_requests: List[int] = []
         finished_reason_requests: List[str] = []
-
+        
+        # Lora requests
+        running_adapters = dict(collectionsCounter([
```

Comment:
Suggest rename to `xxx_lora_adapters` to be more clear.

Reply:
Thanks for the tip, i was debating between adapter and putting in lora.  updated throughout